experimental/casemapping/src/casemapping.rs

```diff
@@ -62,18 +62,11 @@ impl CaseMapping {
         exceptions: &[u16],
         unfold: &[u16],
     ) -> Result<Self, Error> {
-        let internals = CaseMappingV1 {
-            casemap: CaseMappingInternals::try_from_icu(
-                trie_header,
-                trie_index,
-                trie_data,
-                exceptions,
-                unfold,
-            )?,
-        };
+        let data =
```

Comment:
Remove this method? Datagen doesn't need it, and it should not be in the rustdoc.

Reply:
yeah i was wondering why it exists.